app/javascript/modules/listingDetails/ListingDetailsReservedBanner.scss

```diff
@@ -0,0 +1,6 @@
+.educator1-banner {
+  @apply normal-case;
+  @apply space-y-1;
+  @apply tracking-tight;
+  font-size: 13px;
```

Comment:
The convention in our code `scss` files is to use either `rem` or `bloom-font-size` variables to for font sizes.